.circleci/config.yml

```diff
@@ -25,68 +25,6 @@ jobs:
             yarn ci-check
             yarn lerna run ci-check
 
-  deploy:
```

Comment:
@photodow you will need to remove `deploy` under workflows too.

Reply:
Ok, awesome. I think I took care of it. Give it a second look and let me know.